product/dialogs/service_dialogs/transform-vm.yml

```diff
@@ -0,0 +1,221 @@
+---
+- description: Transform VM
+  buttons: submit,cancel
+  label: Transform VM
+  blueprint_id:
+  dialog_tabs:
+  - description:
+    display: edit
+    label: New tab
+    display_method:
+    display_method_options:
+    position: 0
+    dialog_groups:
+    - description: Import given virtual machine to another infrastructure provider
+      display: edit
+      label: Transform Virtual Machine
+      display_method:
+      display_method_options:
+      position: 0
+      dialog_fields:
+      - name: name
+        description: Name of the created virtual machine
```

Comment:
@mkanoor It is the name of the newly created VM in the destination provider. What might be helpful to the user is to pre-fill it with some default value (same as original, add `_transformed` to the original name, ....)